package/greposcope

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: GPL-2.0
+
+# Dependencies:
+# rebuilderd-tools
+# parallel
+
+progname="greposcope"
+tmp_dir="${TMPDIR:-/tmp}/${progname}-${UID}"
+rebuilderd_host="https://reproducible.archlinux.org"
+
+help() {
+	cat <<EOF
+Usage: ${progname} [OPTIONS] pattern
+
+Download and search for 'pattern' in diffoscope outputs of every unreproducible packages reported at ${rebuilderd_host}.
+This is useful to identify packages that are unreproducible because of a specific issue.
+
+OPTIONS
+    -h, --help            Show this message
+    -i, --ignore-case     Ignore case distinctions in patterns and input data
+
+Examples:
+    $ ${progname} "gzip compressed data"
+    $ ${progname} -i zipinfo
+EOF
+}
+
+while ((${#})); do
+	case "${1}" in
+		-h|--help)
+			help
+			exit 0
+		;;
+		-i|--ignore-case)
+			extra_grep_opt="--ignore-case"
+		;;
+		--)
+			shift
+			break
+		;;
+		-*)
+			echo -e >&2 "Invalid argument -- '${1}'\nTry '${progname} --help' for more information"
+			exit 1
+		;;
+		*)
+			pattern="${1}"
+		;;
+	esac
+	shift
+done
+
+if [ -z "${pattern}" ]; then
+    echo -e >&2 "No pattern provided\nTry '${progname} --help' for more information"
+    exit 1
+fi
+
+mkdir -p "${tmp_dir}"
```

Comment:
things could get wonkey as you don't have `set -e` and things like this can fail.

Reply:
Indeed, I added `set -eou pipefail` ;) https://github.com/archlinux/contrib/pull/89/commits/96b5f47e325b9c7088608ba994eff9d9079554ad